### PR TITLE
Feature/dx 244b state changes

### PR DESCRIPTION
**Pull request by Velenir**
State changes that come with `sell | buy: TokenCode` to `DefaultTokenObject` change

### CODE REVIEW

File: src/components/AuctionPriceBar/index.tsx [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-  sellToken: TokenCode,
+  sellToken: DefaultTokenObject,
   sellTokenPrice: Balance,
 }
 
 const AuctionPriceBar: React.SFC<AuctionPriceBarProps> = ({ sellToken, sellTokenPrice, buyToken, header }) => 
   <div className="auctionLastPrice">
     <small>{`${header} of last auction:`}</small>
-    <big>{`1 ${sellToken} = ${sellTokenPrice} ${buyToken}`}</big>
+    <big>{`1 ${sellToken.symbol || sellToken.name || sellToken.address} = ${sellTokenPrice} ${buyToken.symbol || buyToken.name || buyToken.address}`}</big>
```
Comment (W3stside) [inline src/components/AuctionPriceBar/index.tsx]: could consider passing these as a prop `sellTokenName = buyToken.symbol || buyToken.name || buyToken.address` or similar

File: src/reducers/tokenPair.ts [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
-import { TokenItemProps } from 'components/TokenItem'
+import { TokenPair, DefaultTokenObject, TokenMod } from 'types'
 
-export default handleActions<TokenPair & { sellAmount: Balance }, TokenItemProps & TokenPair>(
+// TODO: fill in state with dispatch in or around getDefaultTokens
+const initialState: TokenPair = {
+  sell: {
+    name: "ETHER",
+    symbol: "ETH",
+    address: "0x254dffcd3277c0b1660f6d42efbb754edababc2b",
```
Comment (W3stside) [inline src/reducers/tokenPair.ts]: @Velenir from `truffle network` - this is TokenGNO and your TokenGNO = RDN. Just change these to:
```
  sell: {
    name: "ETHER",
    symbol: "ETH",
    address: "0xcfeb869f69431e42cdb54a4f4f105c19c080a601",
    decimals: 18,
  },
  buy: {
    name: "GNOSIS",
    symbol: "GNO",
    address: "0x254dffcd3277c0b1660f6d42efbb754edababc2b",
    decimals: 18,
  },
```

Comment (Velenir) [reply src/reducers/tokenPair.ts]: good catch

File: src/components/AuctionPriceBar/index.tsx (hunk 1 above)
Comment (Velenir) [reply src/components/AuctionPriceBar/index.tsx]: Good idea, won't be able to do it everywhere as sometimes the whole object is necessary, but will wherever possible.

**Review by W3stside — APPROVED** [review]
After 2 PR merges, approving.

**Pull request merged**